documentation/content/el/books/handbook/desktop/_index.adoc

```diff
@@ -175,7 +175,7 @@ endif::[]
 
 [source,shell]
 ....
-# pkg_add -r opera
+# pkg install opera
```

Comment:
`opera` does not exist. Instead, maybe `linux-opera`

<https://www.freshports.org/www/linux-opera/>

Reply:
yes the handbook says replace opera with linux-opera in the following paragraph